waku/node/v2/wakunode2.nim

```diff
@@ -163,6 +166,36 @@ proc subscribe*(w: WakuNode, contentFilter: ContentFilter, handler: ContentFilte
   # TODO: get some random id, or use the Filter directly as key
   w.filters.add("some random id", Filter(contentFilter: contentFilter, handler: handler))
 
+# TODO Move these helpers up, here to make it easier to iterate on API
+let defaultTopic = cast[Topic]("waku")
+let emptyContentFilter = ContentFilter()
+# TODO Name these functions to ensure type interference works correctly?
+proc emptyTopicHandler*(topic: string, data: seq[byte]) {.async.} = discard
+proc emptyContentFilterHandler*(message: seq[byte]) = discard
+
+proc subscribe2*(node: WakuNode,
+                 topic = defaultTopic,
+                 topicHandler = emptyTopicHandler,
```

Comment:
I think it would be better set to `defaultTopicHandler` instead of an empty one. Just not to allow misuse. And inside a check should be done if there is already such same subscription.

Still looks quite confusing as API though, anyhow, I think we can focus on this better once we understand if it is really needed to set multiple pubsub topics and for what. 
Imo more important right now is to get the basics of the different protocols working and integrated in the WakuNode.

Reply:
I thought about that, but thought that might make more sense at the `status-waku` library level. Or do you think this should be default for all people using Waku? I guess that could be fine.

Yeah agree it is a bit confusing, I'd be fine leaving this with the two calls and we can improve on it later.